.travis/build-site-v6

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# This builds the new v6 website documentation and apps
+#
+
+set -o errexit -o nounset
+
+exe() { echo "\$ ${@/eval/}" ; "$@" ; }
+
+TARGET=deploy/$(cat ./version.txt)
+mkdir -p $TARGET
+
+[ -d ./node_modules/@qooxdoo/compiler ] || npm i --no-save @qooxdoo/compiler
+
+# build apps (unless already built) & copy to website target
+[ -d ./apps/resource ] || exe npx qx compile compile.json --target build  --feedback=0
+exe cp -a ./apps $TARGET
+exe find $TARGET -type f -name '*.map' -delete
```

Comment:
Should'nt we put this into a compile.js file so that it works for npm_deploy too?

Reply:
Yes, this could be DRYed up with `$TARGET` being a CLI parameter - but let's test it "live" first, then optimize later.

Reply:
aggreed